### PR TITLE
Update testing-your-ssh-connection.md

### PR DESCRIPTION
**Pull request by charlescurley**
The edited language was confusing and incorrect. See https://github.community/t/ssh-fingerprint-matching-issue/150693

### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by syamilcobra — APPROVED** [review]

**Comment by janiceilene** [comment]
@charlescurley Thanks so much for opening a PR! I'll get this triaged for review 🌟

**Comment by charlescurley** [comment]
I get email....

This PR is stale because it has been open 7 days with no activity and will be automatically closed in 3 days. To keep this PR open, update the PR by adding a comment or pushing a commit.

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

**Review by hubwriter — APPROVED** [review]
@charlescurley - Great correction! Many thanks for raising this PR, and apologies for our delay reviewing it over the holiday period.

File: content/github/authenticating-to-github/testing-your-ssh-connection.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ When you test your connection, you'll need to authenticate this action using you
   > Are you sure you want to continue connecting (yes/no)?
   ```
 
-3. Verify that the fingerprint in the message you see matches one of the messages in step 2, then type `yes`:
+3. Verify that the fingerprint in the message you see matches one of the [Github fingerprints](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/githubs-ssh-key-fingerprints). If it does, then type `yes`:
```
Comment (hubwriter) [inline content/github/authenticating-to-github/testing-your-ssh-connection.md:34]:
```suggestion
3. Verify that the fingerprint in the message you see matches one of the [Github fingerprints](/github/authenticating-to-github/githubs-ssh-key-fingerprints). If it does, then type `yes`:
```

**Pull request merged**